main_logic/core.py

```
@@ -799,7 +801,13 @@ def _respawn_tts_worker(self):
         import time
         now = time.monotonic()
         if now - self._last_tts_respawn_time < 12.0:
-            return  # 冷却中，跳过
+            return  # 冷却中，保留待执行的延迟任务和错误码状态
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)